### DISCUSSION TITLE
How to add LLaVA 1.5 to Bubble.io ?

### DISCUSSION THREAD

**Discussion by Wims668** [haotian-liu/LLaVA#547] [Q&A]
I'm writing to you because I'd like to create a program that can analyze images imported into the same program, so that the latter can draw conclusions.
I'd like to insert LLaVA on Bubble.io on any application creation site.
Is this possible?
If so, how do I go about it?
Could you please help me?
I love LLaVA and this program inspires me enormously.
So thank you!

Have a nice day.

Best regards,

Willie WIMS

**Comment by haotian-liu** [comment]
For these integrations, you may try something like this to embed LLaVA into your code: https://github.com/haotian-liu/LLaVA/blob/main/llava/eval/run_llava.py
